src/NServiceBus.Persistence.DynamoDB/Outbox/OutboxPersister.cs

```diff
@@ -90,7 +110,7 @@ public OutboxPersister(IAmazonDynamoDB dynamoDbClient, OutboxPersistenceConfigur
         ContextBag contextBag)
     {
         // Using numberOfTransportOperations instead of transportOperationsAttributes.Count to account for
-        // potential partial deletes
+        // potential partial deletes/phantom records
```

Comment:
I think there should be no phantom records in here?

Reply:
Removed the comment